tests/conftest.py

```diff
@@ -990,6 +990,15 @@ def closed_review_minimal_community(minimal_community):
     return community
 
 
+@pytest.fixture()
+def closed_submission_minimal_community(minimal_community):
+    """Data for a minimal community that restricts record submission."""
+    community = deepcopy(minimal_community)
+    community["slug"] = "closed-submission-community"
+    community["access"]["submission_policy"] = "closed"
```

Comment:
it should be changed after the renaming proposed in https://github.com/inveniosoftware/invenio-communities/pull/1189